core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -145,6 +145,14 @@ public static void setupContent() throws FileNotFoundException {
 //    }
 
     @Test
+    public void testIsRunning() {
+        GenericContainer container = new GenericContainer();
```

Comment:
please add `try(GenericContainer container = new GenericContainer()) {` here (it implements `AutoClosable`), otherwise the container will remain until JVMs shutdown.